pkg/server/status/recorder.go

```diff
@@ -609,8 +609,9 @@ type registryRecorder struct {
 func extractValue(name string, mtr interface{}, fn func(string, float64)) error {
 	switch mtr := mtr.(type) {
 	case metric.WindowedHistogram:
+		// Use cumulative count here
+		fn(name+"-count", float64(mtr.TotalCount()))
 		n := float64(mtr.TotalCountWindowed())
-		fn(name+"-count", n)
 		avg := mtr.TotalSumWindowed() / n
```

Comment:
Sorry for not pointing this out earlier, but we have a similar but more convoluted problem with `TotalSumWindowed`. Normally histograms export total sum and total count, so that you can take rates and compute local averages. Here we can't do that because while we now have a total count, we don't have a total sum but just a "current average". Could we also export `-sum` which has the `TotalSum()`? At that point it would be nice to not write the prometheus histogram again, so perhaps instead of `TotalCount()` we'd have `.Total() (count uint64, sum float64)`.